internal/testutil/test_schemas.go

```diff
@@ -396,34 +396,46 @@ const RResourcePortLoa = `resource "packetfabric_port_loa" "%s" {
 
 // End of resources templates for required fields only
 
+// Datasource: packetfabric_locations_cloud
 const DDataSourceLocationsCloud = `data "packetfabric_locations_cloud" "%s" {
   provider              = packetfabric
   cloud_provider        = "%s"
   cloud_connection_type = "%s"
 }`
 
+// Datasource: packetfabric_locations_port_availability
 const DDataSourceLocationsPortAvailability = `data "packetfabric_locations_port_availability" "%s" {
   provider  = packetfabric
   pop       = "%s"
 }`
 
+// Datasource: packetfabric_locations
 const DDatasourceLocations = `data "packetfabric_locations" "%s" {
   provider  = packetfabric
 }`
 
+// Datasource: packetfabric_locations_pop_zones
 const DDatasourceLocationsPopZones = `data "packetfabric_locations_pop_zones" "%s" {
   provider = packetfabric
   pop      = "%s"
 }`
 
+// Datasource: packetfabric_locations_regions
 const DDataSourceLocationsRegions = `data "packetfabric_locations_regions" "%s" {
   provider = packetfabric
 }`
 
+// Datasource: packetfabric_activitylog
 const DDatasourceActivityLog = `data "packetfabric_activitylog" "%s" {
   provider = packetfabric
 }`
 
+// Datasource: packetfabric_locations_markets
 const DDataSourceLocationsMarkets = `data "packetfabric_locations_markets" "%s" {
   provider = packetfabric
 }`
+
+// Datasource: packetfabric_ports
+const DDataSourcePorts = `data "packetfabric_ports" "%s" {
```

Comment:
I wonder if we need to add wait before running the data source test?

I am getting this:
```
=== CONT  TestAccDataSourcePortsComputedRequiredFields
    datasource_ports_test.go:15: Step 1/1 error: Check failed: Check 14/31 error: data.packetfabric_ports.terraform_testacc_9da58fce_889d: Attribute 'interfaces.0.operational_status' expected to be set
--- FAIL: TestAccDataSourcePortsComputedRequiredFields (252.62s)
```

But when I check in the API `operational_status: "up"` and things look good in the datasource definition.

Reply:
Let's only test fields which are always set in that case to keep things simple.